parquet/src/test/java/org/apache/iceberg/parquet/TestDictionaryRowGroupFilter.java

```diff
@@ -360,6 +362,21 @@ public void testNotNaNs() {
     Assert.assertTrue("Should read: no_nans column will contain non-NaN", shouldRead);
   }
 
+  @Test
+  public void testNotNaNOnNaNsAndNulls() {
+    boolean shouldRead =
+        new ParquetDictionaryRowGroupFilter(SCHEMA, isNull("_nans_and_nulls"))
```

Comment:
Can you add cases for notNull and notNaN as well?

Reply:
Added tests.